lighthouse-cli/run.js

```diff
@@ -7,22 +7,22 @@
 
 /* eslint-disable no-console */
 
-const path = require('path');
-const psList = require('ps-list');
+/** @typedef {Error & {code: string, friendlyMessage?: string}} ExitError */
```

Comment:
what did you think about my `typedef` after imports thing from https://github.com/GoogleChrome/lighthouse/pull/13046#discussion_r707507056?

Reply:
I prefer custom types at the top of the file, but not enough to discuss it much. I'm reverting the changes that were just moving things around.

Reply:
ha, damn, I thought I had convinced you, but oh well :)

Reply:
the short of it is that exactly _what_ is being imported isn't too informing (and sometimes the import list is so long you glaze over all of it like it's a Java file); but the types are quite useful as to the data structures involved in the file. so for the same reason that fileoverview would be at top, I like local types to be too

Reply:
For all the same reasons I think they should be closer to the code that uses them :) Definitions, just in the type domain instead of the value domain, so should be right next to the value definitions.